### DISCUSSION TITLE
`motelist` argument to `make` does not work for `TARGET=nrf52840`

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2577] [General]
The following goes not work
```
make TARGET=nrf52840 motelist
make: *** No rule to make target 'motelist'.  Stop.
```

Also note, that the following command lists two motes, while only one is connected. I presume that `/dev/ttyACM0` is the mote address and `/dev/ttyACM1` may be its Flash storage?
```
make TARGET=nrf52840 motelist-all
Port          Serial        VID     PID     Product  Vendor
------------  ------------  ------  ------  -------  ------
/dev/ttyACM0  0x1050232358  0x1366  0x1051  J-Link   SEGGER
/dev/ttyACM1  0x1050232358  0x1366  0x1051  J-Link   SEGGER
```

**Comment by pjonsson** [comment]
That line in the documentation is from 2018, something has probably changed in the build system after that. You've found the motelist-all target that I use though.

I think the second port is the stuff mentioned at https://docs.contiki-ng.org/en/develop/doc/platforms/nrf52840.html#virtual-com-and-real-time-transfer.

**Reply by rahav** [reply]
Thanks, and you are probably correct because in the link you provided `/dev/ttyACM0` is used to connect to the mote over a serial port.

My observation was made in a genuine effort to promote Contiki-NG, by making it easier for researchers and developers to use it. As a person outside of the core team it feels that we can do it with many easy wins. In this case one sentence would do it. I would write the sentence or paragraph myself, if I was certain.

Botton line, I would like to see Contiki rise to the top in the figure below.

![OS](https://github.com/contiki-ng/contiki-ng/assets/2912950/70029dcf-852c-4dab-aa27-64f569609f4d)
